internal/authentication/authentication.go

```diff
@@ -12,7 +12,7 @@ const TokenCookieName = "token"
 
 var (
 	NotAuthenticatedError   = errors.New("User does not have a valid authentication token")
-	InvalidCredientalsError = errors.New("Invalid credientals for creating an authentication token")
+	InvalidCredientalsError = errors.New("Invalid credentials for creating an authentication token")
```

Comment:
NIT: If we're fixing the spelling mistake in the error message, could we possibly fix it in the name of the error too?